app/views/static_pages/verification_sent.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= page_heading 'Verification sent' %>
+
+<p>A verification email has been sent to <%= current_user.email_addresses.first.try(:value) %>.  If you don't see the message in your inbox, check your junk or spam folder.</p>
```

Comment:
Can you add a button to resend the verification? (you may already be planning this in a later PR, which is fine).